docs/simpleinjector/metricsCapturing/README.md

```diff
@@ -0,0 +1,10 @@
+# Adding Metrics and Exception Logging With `Functional.CQS.AOP.IoC.SimpleInjector.MetricsCapturing`
+
+`Functional.CQS.AOP.IoC.PureDI.MetricsCapturing` supplies metrics-capturing decorator implementations, while `Functional.CQS.AOP.IoC.SimpleInjector.MetricsCapturing` supplies a set of extension methods that make it easy to register that decoration with the `SimpleInjector` container.
```

Comment:
I wasn't 100% sure if "that decoration" flowed right here. Maybe "those decorations" or "those decorators"?

Reply:
Went with "those decorators"